SCMS.Services.Api/Models/Foundations/Emails/Email.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) Signature Chess Club & MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using SCMS.Services.Api.Models.Foundations.Guardians;
+using SCMS.Services.Api.Models.Foundations.Users;
+using System;
+
+namespace SCMS.Services.Api.Models.Foundations.Emails
+{
+    public class Email
+    {
+        public Guid Id { get; set; }
+        public string Address { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+
+        public Guid GuardianId { get; set; }
```

Comment:
We need establish the relation to the Guardian table.

Reply:
I hope its done by now.